docs/operations.md

```diff
@@ -123,6 +165,110 @@ stays in the database:
 
 **Default**: `30` (days)
 
+## Database option
+
+### Always keep scavenged (_Deprecation Notice_)
+
+This option ensures that the newer chunk from a scavenge operation is always kept.
+
+| Format               | Syntax                    |
+|:---------------------|:--------------------------|
+| Command line         | `--always-keep-scavenged` |
+| YAML                 | `AlwaysKeepScavenged`     |
+| Environment variable | N/A                       |
+
+### Scavenge backend page size
+
+Specify the page size of the scavenge database.The default value is 16KiB.
+
+| Format               | Syntax                         |
+|:---------------------|:-------------------------------|
+| Command line         | `--scavenge-backend-page-size` |
+| YAML                 | `ScavengeBackendPageSize`      |
+| Environment variable | N/A                            | 
+
+### Scavenge backend cache size
+
+Specify the amount of memory, in bytes, to use for backend caching during scavenging. The default value is 64MiB.
+
+| Format               | Syntax                          |
+|:---------------------|:--------------------------------|
+| Command line         | `--scavenge-backend-cache-size` |
+| YAML                 | `ScavengeBackendCacheSize`      |
+| Environment variable | N/A                             |                       
+
+### Scavenge hash users cache capacity
+
+Specify the number of stream hashes to remember when checking for collisions.
+
+| Format               | Syntax                                 |
+|:---------------------|:---------------------------------------|
+| Command line         | `--scavenge-hash-users-cache-capacity` |
+| YAML                 | `ScavengeHashUsersCacheCapacity`       |
+| Environment variable | N/A                                    |
+
+## HTTP option
+
+### Start From Chunk (_Deprecation Notice_)
+
+### Thread
+
+Specify the number of threads to use for running the scavenging process. The default value is 1.
+
+_Thread must be 1 or above_
+
+```
+Ex: POST to http://127.0.0.1:2113/admin/scavenge?thread=2
+```
+
+### Threshold
+
+By default, when running a scavenge, all chunks related to the events that need to be scavenged are removed, and new chunks are created. Setting this option allows you to optimize the chunk weight threshold for scavenging. This optimization helps in resource and time efficiency.
```

Comment:
add:

If you find that scavenge is spending a long time scavenging chunks without removing much information from them, you can provide a threshold greater than zero. The weights of the chunks that are being scavenged or skipped can be found in the log files by searching for "with weight". The weight of a chunk is approximately double the number of records that can be removed from that chunk.